Src/IronPython/Compiler/Tokenizer.cs

```diff
@@ -127,17 +126,6 @@ internal Tokenizer(ErrorSink errorSink, PythonCompilerOptions options, bool verb
             }
         }
 
-        public Severity IndentationInconsistencySeverity {
-            get { return _indentationInconsistencySeverity; }
-            set {
-                _indentationInconsistencySeverity = value;
-
-                if (value != Severity.Ignore && _state.IndentFormat == null) {
-                    _state.IndentFormat = new StringBuilder[MaxIndent];
```

Comment:
Seeing a bunch of
> SystemError: Object reference not set to an instance of an object

in the CI log. Probably need to move this initialization to the constructor.